cscs-checks/prgenv/helloworld.py

```diff
@@ -220,4 +225,4 @@ def __init__(self, lang, linkage):
         # num_cpus_per_task, but adding for completeness and portability
         self.variables = {
             'OMP_NUM_THREADS': str(self.num_cpus_per_task)
-        }
+        }
```

Comment:
line 117: this `PrgEnf-pgi` looks like a typo

Reply:
line 31: you may want to remove `PrgEnv-cray_classic`as it is no longer part of `config/cscs.py`